dbt/tests/data_tests/generic_tests/expect_complete_valid_ownership.sql

```diff
@@ -0,0 +1,38 @@
+{% test expect_complete_valid_ownership(model, n_acceptable_failures=0) %}
+with OwnSum as (
+    select
+        report_date,
+        plant_id_eia,
+        generator_id,
+        sum(fraction_owned) as fraction_owned,
+        1 as toy_join
+    from {{ model }}
+    group by
+        report_date,
+        plant_id_eia,
+        generator_id
+    having fraction_owned not null
+), OwnSumInvalid as (
+    select * from OwnSum where fraction_owned > 1.02
```

Comment:
@krivard What is the significance of the value 1.02 here? It looks cleaner to me if that is a parameter

Reply:
[It was hard-coded in the source test](https://github.com/catalyst-cooperative/pudl/blob/f564cd04e6e0e8512bcf026718bf7d8fe5bf8f8d/test/validate/eia860_test.py#L57); @zaneselvans do you have insight here as the last one to edit those lines? Should this and the 0.98 below remain fixed or move to input parameters?

Reply:
I think it would be ideal if they were parameters, but for time's sake I think its ok if they are hard coded. If we want to use this function on another table and use a different parameter we can tweak it then.